.template/addons/github/.github/workflows/test.yml.tt

```diff
@@ -0,0 +1,76 @@
+name: Test
+
+on: push
+
+env:
+  DOCKER_REGISTRY: ${{ secrets.DOCKER_REGISTRY }}
+  DOCKER_USERNAME: ${{ github.repository_owner }}
+  DOCKER_TOKEN: ${{ secrets.DOCKER_TOKEN }}
+  COMPOSE_FILE: docker-compose.test.yml
+  DOCKER_IMAGE: ${{ github.repository }}
+
+jobs:
+  build:
+    name: Build docker image
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2.3.2
+
+      - uses: nimblehq/branch-tag-action@v1.1
+
+      - name: Docker login
+        run: echo "$DOCKER_TOKEN" | docker login $DOCKER_REGISTRY --username=$DOCKER_USERNAME --password-stdin
+
+      - name: Docker pull
+        if: env.BRANCH_TAG != 'latest' && env.BRANCH_TAG != 'development'
+        run: docker-compose pull test || true
+
+      - name: Docker build
+        run: |
+          bin/docker-prepare
+          docker-compose build
+          docker-compose push test
+
+  unit_test:
+    name: Unit test
+    runs-on: ubuntu-latest
+    needs: build
+    steps:
+      - name: Cancel previous runs
+        uses: styfle/cancel-workflow-action@0.5.0
+        with:
+          access_token: ${{ github.token }}
+
+      - uses: actions/checkout@v2.3.2
+
+      - uses: nimblehq/branch-tag-action@v1.1
+
+      - name: Docker login
+        run: echo "$DOCKER_TOKEN" | docker login $DOCKER_REGISTRY --username=$DOCKER_USERNAME --password-stdin
+
+      - name: Docker pull
+        run: docker-compose pull
+
+      - name: Linters
+        run: docker-compose run test yarn lint
```

Comment:
As I remember `yarn lint` was execute in the `codebase_spec.rb` already

Reply:
let me check again

Reply:
Yes, it will be run in `codebase_spec.rb`.
I removed it